client/camera/package.json

```diff
@@ -34,28 +34,28 @@
     "branches": 80
   },
   "devDependencies": {
-    "@types/bluebird": "^3.5.8",
-    "@types/chai": "^4.0.3",
+    "@types/bluebird": "^3.5.18",
```

Comment:
I think that using `^` is the main source of all our compilation problems. 
Not locking types brings some incompatibilities across the board. If we do not want to go back to shrinkwrap best to lock them - possibly separate ticket.

Reply:
It seems `npm-check-updates` has a --removeRange flag, I'll give it a try.

Reply:
Ideally we could start including the new npm 5+ package-locks in version control too.